src/components/Login.js

```diff
@@ -1,56 +1,56 @@
-import { useState, useEffect, useRef } from "react";
+import { useState } from "react";
+import axios from 'axios'
+import axiosInstance from "../api/axios";
+import { Link, useNavigate } from "react-router-dom";
 
 const Login = () => {
-  const errRef = useRef();
-
-  const [user, setUser ] = useState({username: "", password: ""});
-  const [errMsg, setErrMsg] = useState("");
+  const [user, setUser] = useState({ username: "", password: "" });
+  const [errLog, setErrLog] = useState(null);
   const [success, setSuccess] = useState(false);
 
+  let navigate = useNavigate();
+
   const handleSubmit = async (e) => {
     e.preventDefault();
-    setUser("");
-    setSuccess(true);
-  };
+    try {
+      const result = await axiosInstance.post("token/", user)
+      console.log(result.data);
+      localStorage.setItem("user-data", JSON.stringify(result.data));
```

Comment:
Why not a state variable?

Reply:
I am still trying to get and understand this login the way I want to, there are different ways this could end up but my goal is to get it right.